cookbook/gestures/dismissible.md

```diff
@@ -9,13 +9,13 @@ we're writing an email app, we might want to allow our users to swipe away email
 messages in a list. When they do, we'll want to move the item from the Inbox to 
 the Trash.
 
-Flutter makes this task easy by providing the [`Dismissable`](https://docs.flutter.io/flutter/widgets/Dismissible-class.html)
+Flutter makes this task easy by providing the [`Dismissible`](https://docs.flutter.io/flutter/widgets/Dismissible-class.html)
 Widget.
 
 ## Directions
 
   1. Create List of Items
-  2. Wrap each item in a `Dismissable` Widget
```

Comment:
Haha, I thought I had fixed all of these! 😭

Reply:
I fixed all the ones I found grep!